sycl/source/detail/force_device.cpp

```diff
@@ -10,6 +10,7 @@
 #include <CL/sycl/info/info_desc.hpp>
 #include <CL/sycl/stl.hpp>
 #include <cstdlib>
+#include <algorithm>
```

Comment:
Please, sort header and separate standard header by an empty line.

Reply:
Done.

Reply:
I think `#include <algorithm>` should go before `#include <cstdlib>`.